### PR TITLE
#2 Add table example

### PR DESCRIPTION
**Pull request by danielrus**

### CODE REVIEW

**Review by ZorckMaster — APPROVED** [review]

**Comment by alexandrudanpop** [comment]
Can you please link the associated issue?

File: src/app/tasks/tasks.module.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 
 import { TasksRoutingModule } from './tasks-routing.module';
 import { TasksComponent } from './tasks.component';
+import { MaterialModule } from '../material.module';
```
Comment (alexandrudanpop) [inline src/app/tasks/tasks.module.ts:7]: please import the SharedModule. The SharedModule already imports the MaterialModule. 
Dependency graph between modules should be like:
image

**Review by alexandrudanpop — CHANGES_REQUESTED** [review]

**Review by alexandrudanpop — DISMISSED** [review]

**Review by alexandrudanpop — APPROVED** [review]

File: src/app/tasks/tasks.component.scss [hunk 2]
```diff
@@ -1 +1,45 @@
...
+  margin-right: 1rem;
+}
+
+.overlay {
+  position: absolute;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 56px;
+  background: rgba(0, 0, 0, 0.15);
```
Comment (ZorckMaster) [inline src/app/tasks/tasks.component.scss:40]: We could use a variable for black color

File: src/app/tasks/shared/tasks.service.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    pageNumber?: number
+  ): Promise<TaskApi> {
+    let results: Task[] = [...TASKS];
+
+    if (sortKey && order) {
+      results = results.sort((a: Task, b: Task) => {
+        if (a[sortKey] > b[sortKey]) {
+          return order === 'asc' ? 1 : -1;
+        }
+        if (b[sortKey] > a[sortKey]) {
```
Comment (ZorckMaster) [inline src/app/tasks/shared/tasks.service.ts:22]: 'asc' could be added to a constant, and use the constant everywhere is needed.

File: src/app/tasks/shared/task.model.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+export interface Task {
+  id: string;
+  associatedLink: string;
+  context: string;
+  type: string;
+  region: string;
+  district: string;
+  priority: string;
+  startDate: string;
```
Comment (ZorckMaster) [inline src/app/tasks/shared/task.model.ts:9]: Shouldn't these be of type: Date ?

File: src/app/tasks/tasks.component.scss (hunk 2 above)
Comment (danielrus) [reply src/app/tasks/tasks.component.scss:40]: i think we should do this after a design is approved

File: src/app/tasks/shared/tasks.service.ts (hunk 3 above)
Comment (danielrus) [reply src/app/tasks/shared/tasks.service.ts:24]: updated

File: src/app/tasks/shared/task.model.ts (hunk 4 above)
Comment (danielrus) [reply src/app/tasks/shared/task.model.ts:9]: it's just example mock data, it has no impact

**Review by ZorckMaster — APPROVED** [review]

**Pull request merged**